fcli-core/fcli-sc-sast/src/main/resources/com/fortify/cli/sc_sast/i18n/SCSastMessages.properties

```diff
@@ -74,6 +74,7 @@ fcli.sc-sast.session.login.ssc-url.0 = SSC URL.
 fcli.sc-sast.session.login.ssc-url.1 = Environment variables:%n \
   ${fcli.env.default.prefix}_SSC_URL: Shared with SSC/SC DAST%n \
   ${fcli.env.default.prefix}_SC_SAST_SSC_URL: Only SC SAST commands
+fcli.sc-sast.session.login.ctrl-url = Scan Central Controller URL.%n\By default the Controller URL shall be fetched from the SSC configuration, however with the use of this option, the user defined controller URL overrides the one fetched from SSC Configuration.
```

Comment:
Consider shortening this to something like `Override ScanCentral SAST Controller URL. If not specified, the controller URL as configured in SSC will be used.`